tests/cupy_tests/linalg_tests/test_decomposition.py

```diff
@@ -93,6 +93,13 @@ def test_mode(self):
         self.check_mode(numpy.random.randn(3, 3), mode=self.mode)
         self.check_mode(numpy.random.randn(5, 4), mode=self.mode)
 
+    @testing.fix_random()
+    @condition.repeat(3, 10)
```

Comment:
There's no randomness in the inputs.  Could you remove `fix_random` and `repeat`?  `numpy.empty((0, 3))` can be used below.